src/stories/tasks/TasksTable.stories.tsx

```diff
@@ -11,30 +11,27 @@ import {
   sampleGitHubUser2,
   sampleGitHubUser3,
   sampleTask1,
+  sampleTask2,
+  sampleTask3,
+  sampleTask4,
+  sampleTask5,
 } from '../fixtures';
 
 export default {
   title: 'Tasks/Table/Component',
   component: TasksTableComponent,
-  decorators: [
-    withRedux({
-      user: {
-        id: 'user-id',
-        username: 'currentUser',
-      },
-    }),
-  ],
+  decorators: [withRedux()],
 };
 
 const Template: Story<ComponentProps<typeof TasksTableComponent>> = (args) => (
   <TasksTableComponent {...args} />
 );
 
-export const TasksTable = Template.bind({});
-TasksTable.args = {
+export const Component = Template.bind({});
+Component.args = {
   projectSlug: 'my-project',
   epicSlug: sampleEpic1.slug,
-  tasks: [sampleTask1],
+  tasks: [sampleTask1, sampleTask2, sampleTask3, sampleTask4, sampleTask5],
```

Comment:
@duggiemitchell @jgerigmeyer  how do these get ordered. By status? 
![Screen Shot 2021-01-28 at 8 59 14 AM](https://user-images.githubusercontent.com/1581694/106156520-3eda5380-6147-11eb-8ccf-f0a24003aea0.jpg)

Reply:
@stacyk Ah, right. First "In Progress" (which includes "Approved" and "Changes Requested" all together), then "Planned", then "Complete". Within categories, sorting is alphabetical by name.

Reply:
Excellent! Thanks